docs/components/receive.md

```diff
@@ -77,6 +77,20 @@ The example content of `hashring.json`:
 
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed for tenancy and replication.
 
+## Limiting
+
+Thanos Receive, in Router mode, supports limiting tenant active series, to maintain stability of the system. It uses any Prometheus Query API compatible meta-monitoring solution to get the current number of active series, and compares that with a configured limit, before ingesting any tenant's remote write request. In case a tenant has gone above the limit, their remote write requests are failed fully.
```

Comment:
Do you mean we should mention something like "meta-monitoring should scrape all Thanos Receive instances"? 🙂 
Will add that!

But as for how to set up meta-monitoring solution in general, I think that would vary by use case.

Reply:
"Meta-monitoring" under the Thanos docs [Operating](https://thanos.io/tip/operating/) section would be a good idea, and it could specify how to set up meta-monitoring for different Thanos architectures. This would probably need a dedicated different PR. 🙂

Reply:
Updated in https://github.com/thanos-io/thanos/pull/5520/commits/b647b541e4202fd6c91967fad20f35f2a94a8363